AWESOME.md

```diff
@@ -36,3 +41,180 @@
+Liquid Wiki | Federated qiki ported to Holochain | https://github.com/holochain/fed-wiki | ![GitHub last commit](https://img.shields.io/github/last-commit/holochain/fed-wiki.svg)
+Omni | Open Access scholarly communications platform with fully p2p article review system | https://github.com/OmniProject/omni | ![GitHub last commit](https://img.shields.io/github/last-commit/OmniProject/omni.svg)
+QuerySearch (previously HoloDex) | Bridged app for indexing your DHT data | https://github.com/HC-Interns/hc-querysearch | ![GitHub last commit](https://img.shields.io/github/last-commit/HC-Interns/hc-querysearch.svg)
+Personas/Vault | App for people to manage how other apps access their personal information | https://github.com/holochain/holo-vault | ![GitHub last commit](https://img.shields.io/github/last-commit/holochain/holo-vault.svg)
+DPKI (Distributed public key infrastructure) | Key management and self-sovereign identity | https://github.com/holochain/DPKI | ![GitHub last commit](https://img.shields.io/github/last-commit/holochain/DPKI.svg)
+Backups | Back up your source chain entries to rebuild your state in case of data loss | https://github.com/HC-Interns/proto-hc-backup | ![GitHub last commit](https://img.shields.io/github/last-commit/HC-Interns/proto-hc-backup.svg)
+Anchors | Keeping your DHT data "found" | https://github.com/holochain/mixins | ![GitHub last commit](https://img.shields.io/github/last-commit/holochain/mixins.svg)
+PeerVis | A way to visually see who is online and connected as a peer in your Holochain app | https://github.com/holochain/PeerVis | ![GitHub last commit](https://img.shields.io/github/last-commit/holochain/PeerVis.svg)
+Scaffold Tool | Web tool for rapid creation of DNA structure | https://github.com/holochain/scaffold | ![GitHub last commit](https://img.shields.io/github/last-commit/holochain/scaffold.svg)
+DNA-To-OpenAPI | given holochain dna, generate a swagger / openapi spec file | https://github.com/holochain/dna-to-openapi | ![GitHub last commit](https://img.shields.io/github/last-commit/holochain/dna-to-openapi.svg)
+HC-Redux-Middleware | Middleware for Redux.js to provide holochain calls | https://github.com/holochain/hc-redux-middleware | ![GitHub last commit](https://img.shields.io/github/last-commit/holochain/hc-redux-middleware.svg)
+bs-holochain | Holochain Bindings for Bucklescript | https://github.com/struktured/bs-holochain | ![GitHub last commit](https://img.shields.io/github/last-commit/struktured/bs-holochain.svg)
+
+
+## Lost/Orphaned apps
+If any of these apps belong to you please let us know so we can list them with a URL
+ - Trust Graph: Claims, ratings, and reputation
+ - Fileshare: P2P filesharing
+ - Rochambeaux: Rock-Paper-Scissors (demonstrates use of private chain entries)
+ - Agent.Earth: Disaster response
+ - Hackstage: Event mangagement
+ - MyData private data sharing:
+ - Unmute / Medium / P2P Blognet:
+ - P2P Library:
```

Comment:
Perhaps you mean 'MailBoox' by Viktor Zaunders?
https://github.com/zaunders/Mailboox-Rust